app/background.py

```diff
@@ -46,22 +46,39 @@ def put(self, data):
 
 
 def background(inputQueue, outputQueue):
+  def redrawProgram(program):
+    """
+    Sends a SIGUSR1 signal to the current program and draws its screen.
+
+    Args:
+      program (CiProgram): an instance of the CiProgram object.
+
+    Returns:
+      None.
+    """
+    pid = os.getpid()
+    signalNumber = signal.SIGUSR1
```

Comment:
How about moving these above this def (so they are only done once, rather than with each call to redrawProgram)
```
   pid = os.getpid()
   signalNumber = signal.SIGUSR1
```

Reply:
Sounds good!